src/stores/networkStore.ts

```diff
@@ -0,0 +1,77 @@
+import { action, computed, observable } from 'mobx';
+import TransitType from '../enums/transitType';
+
+export class NetworkStore {
+    @observable private _enabledTypes: TransitType[];
```

Comment:
Somewhere in our code we use verb "select" and somewhere "enable". For example we call this variable "selectedTypes" and somewhere "enabledTypes". Pick 1 as a convention. I would personally use "select, selected etc."